### DISCUSSION TITLE
Persistent layouts

### DISCUSSION THREAD

**Discussion by 1derwood** [vercel/next.js#11105] [Help]
A while back @timneutkens alluded to forthcoming guidance from @Timer on the proper way to do persistent layouts (https://twitter.com/timneutkens/status/1185174317883105280). Has that been published somewhere?

**Comment by canrozanes** [comment]
I believe its this:
https://github.com/zeit/next.js/tree/canary/examples/with-dynamic-app-layout

**Reply by aaronjensen** [reply]
Yes, that's probably it, though it appears that this may not work well with future versions of React:

https://github.com/zeit/next.js/pull/11091#issuecomment-599950904
https://github.com/zeit/next.js/pull/11091#issuecomment-621071641

It's unclear what, if any, updated guidance there is.
